config/deploy/stage.rb

```diff
@@ -0,0 +1,6 @@
+ENV['APP_ENV']  ||= 'staging'
+require_relative 'server_settings'
+
+server 'sdr-services-test.stanford.edu', user: 'sdr2service', roles: %w{app}
+
+Capistrano::OneTimeKey.generate_one_time_key!
```

Comment:
The `stage` environment should behave like a `production` system, so the bundle should not deploy and dev/test gems.  We need to keep the 

```
set :bundle_without, %w{development local test}.join(' ')
```